src/commands/generate/handler.js

```diff
@@ -13,6 +13,10 @@ internals.generate = async (url = '', options = {}) => {
   });
   const page = await browser.newPage();
 
+  if (options.viewportSize) {
+    await page.setViewport(options.viewportSize);
```

Comment:
Since we are only passing `height` and `width` here, does this affect the other options for `setViewport`? I'm hoping it will continue using defaults if those options aren't explicitly passed, but I would like to verify.